RecoEgamma/EgammaIsolationAlgos/plugins/EgammaIsoHcalDetIdCollectionProducer.cc

```diff
@@ -17,13 +17,16 @@ EgammaIsoHcalDetIdCollectionProducer::EgammaIsoHcalDetIdCollectionProducer(const
   phosToken_ = 
   	  consumes<reco::PhotonCollection>(iConfig.getParameter< edm::InputTag >("phosLabel"));
 
+  ootPhosToken_ = 
+  	  consumes<reco::PhotonCollection>(iConfig.getParameter< edm::InputTag >("ootPhosLabel"));
```

Comment:
I'm not convinced that this change is needed in the producer.
The IDs are added in a fully factorized way in this implementation.
So, there is no benefit between this and a clone producer.
This will also be much more maintainable if some other kind of non-standard photons or cluster collections are added.

... the same comment applies to the ES.